code/_globalvars/lists/poll_ignore.dm

```diff
@@ -45,8 +45,8 @@ GLOBAL_LIST_INIT(poll_ignore_desc, list(
 	POLL_IGNORE_IMAGINARYFRIEND = "Imaginary Friend",
 	POLL_IGNORE_SPLITPERSONALITY = "Split Personality",
 	POLL_IGNORE_CONTRACTOR_SUPPORT = "Contractor Support Unit",
-	POLL_IGNORE_PAI = "Personal AI",
-	POLL_IGNORE_EXPERIMENTAL_CLONE = "Experimental clone"
+	POLL_IGNORE_PAI = ROLE_KEY_PAI,
+	POLL_IGNORE_EXPERIMENTAL_CLONE = ROLE_KEY_EXPERIMENTAL_CLONE
```

Comment:
these are not real job, so that's why these are `ROLE_` instead of `JOB_`, and the naming `KEY` can be removed actually as there is no actual difference between key and name for ROLE type defines, but I did it to recognise what's changed from bee-master.

Reply:
I meant to say ROLE_NAMEs

Reply:
maybe some days later. I will put DO-TO here. It's not currently needed.